index.js

```diff
@@ -3,146 +3,191 @@ const debug = require("debug")("signalk:signalk-to-nmea2000");
 const util = require("util");
 const toPgn = require("to-n2k").toPgn;
 const _ = require('lodash')
+const path = require('path')
+const fs = require('fs')
 
 module.exports = function(app) {
   var plugin = {};
   var unsubscribes = [];
-  var timer;
+  var timers = []
+  var conversions = load_conversions(app, plugin)
+  conversions = [].concat.apply([], conversions)
+
+  var types = {
```

Comment:
It seems like the conversion types are an essential concept when defining the conversions. I struggled to understand their differences by following the code only. Could you maybe add a some comments clarifying the different conversion types?